model/service_broker/service_broker.go

```diff
@@ -0,0 +1,43 @@
+package model
```

Comment:
This is a SC model I think. Service broker doesn't actually know about itself :)